Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Meshing_dialog.ui

```diff
@@ -359,27 +315,25 @@
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
-        <property name="buddy">
-         <cstring>facetSizing</cstring>
+       </widget>
+      </item>
+      <item row="1" column="1">
+       <widget class="DoubleEdit" name="approx">
+        <property name="text">
+         <string>0,00</string>
         </property>
        </widget>
       </item>
+      <item row="2" column="1">
+       <widget class="DoubleEdit" name="facetSizing"/>
+      </item>
       <item row="3" column="1">
-       <widget class="QDoubleSpinBox" name="facetAngle">
-        <property name="minimumSize">
-         <size>
-          <width>110</width>
-          <height>0</height>
-         </size>
-        </property>
-        <property name="minimum">
-         <double>0.000000000000000</double>
+       <widget class="DoubleEdit" name="facetAngle">
+        <property name="inputMask">
+         <string>25,00</string>
         </property>
-        <property name="maximum">
-         <double>30.000000000000000</double>
-        </property>
-        <property name="value">
-         <double>25.000000000000000</double>
```

Comment:
There was a meaning in the minimum and maximum values. I think the `QDoubleValidator` can implement that.

I stop my review here. There are similar issues below.